rllib/examples/learners/ppo_with_torch_lr_schedulers.py

```diff
@@ -11,8 +11,8 @@
 
 How to run this script
 ----------------------
-`python [script file name].py --enable-new-api-stack --lr-const-factor=0.1
---lr-const-iters=10 --lr-exp-decay=0.3`
+`python [script file name].py --enable-new-api-stack --lr-const-factor=0.9
```

Comment:
Could we add a quick custom callback here that checks the learning rate after each iteration?

Roughly like this:
```
class LRChecker(DefaultCallbacks):
   def on_train_result(self, result, algorithm, ...):
      assert algorithm.learner_group._learner.get_optimizer(DEFAULT_MODULE_ID)... param_groups.... ["lr"] == [some value based on `algorithm.iteration`]
```

It's ok to do this check only for the first 2 or 3 iterations.